vuepress/docs/next/tutorials/samples/customer-portal.md

```diff
@@ -0,0 +1,125 @@
+3. Copy the second command and do the same. The content bundle will now be deployed.
+4. Go to the Entando App Builder and navigate to the Repository.  Here you will see the two bundles for the Portal. Install both bundles in the version you desire. You will be able to update the version at any time.
+5. To navigate to your Customer Portal: <br>
+from the left menu →  `Page` → `Mangement` <br>
+find `Customer Portal` under `Page Tree` list<br>
+from the `Actions` pull-down menu →  `View Published Page`
+
+## Manual Install 
+Top install the Entando Customer Portal manually to your specific environment, go to the [Entando Github Repository](https://github.com/entando-samples/customer-portal/tree/main/application) and refer to the README.MD there for more details.
+
+# Configuration
+## Administrators
+In order to configure the Customer Portal and its users, the administrator will need Jira Service Manager and Entando Identity Management credentials. As the administrator, you can create and configure users and groups, assign roles, projects and track service tickets.  You can also customize the ticketing system. 
+
+### JIRA Service Manager 
+
+The administrator begins here to create users and projects, define the organization, and configure the service ticket system.
+
+Go to Customers to add organizations and projects. Once added, click on the name of the organization to get the ID from the URL which is needed later, e.g example.com/jira/servicedesk/projects/ECS/organization/3, where the organization ID is “3”.
+
+For more information or help with the Jira Service Management System, contact ??????
+
+### Entando Identity Management System 
+Logging into the Entando Identity Management System, you will see the landing page below. The left navigation bar is your guide for managing users, groups, and most importantly, roles for your application. Using the RBAC model, you define what users have access to by the roles and groups they are assigned to. Some important guidelines are noted below. 
+
+![Entando Identity Management System](./images/custportal-idmanagement-main.png)
+
+A. **Define the Realm Setting**. The Realm is a set of users, credentials, roles, and groups. A user belongs to and logs into a Realm. 
+
+B. **Create Roles**\
+You must select the roles available to users and/or groups. Create new roles with varying degrees of access for the needs of your application. You can also use the default roles by clicking on Client Roles and choosing entandodemo-customerportal-server. The following define their parameters for access:
+* cp-customer -  assigned directly to specific projects for a single customer
+* cp-partner - assigned directly to specific projects for multiple customers
+* cp-support -  read only view of all customer projects 
+* cp-admin - admin access for Customer Portal.
+
+C. **Create New Users**:
+1. The user must be created in Jira first, then configured here for access to the Portal.
```

Comment:
would nix "here"
this isn't really a step... introduce the steps with this sentence?